diff --git a/dist/index.js b/dist/index.js
index 135cbffb51236e54eb3d17b2e8a9a26ff151b91c..e5c2a7286eacf2b0e580a6c23e1f520ffc772ce1 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -8330,7 +8330,8 @@ function SignInForm({
     toast,
     Link,
     localizeErrors,
-    emailVerification
+    emailVerification,
+    passkey,
   } = useContext56(AuthUIContext);
   const rememberMeEnabled = credentials == null ? void 0 : credentials.rememberMe;
   const usernameEnabled = credentials == null ? void 0 : credentials.username;
@@ -8437,7 +8438,7 @@ function SignInForm({
               /* @__PURE__ */ jsx75(FormControl, { children: /* @__PURE__ */ jsx75(
                 Input,
                 {
-                  autoComplete: usernameEnabled ? "username" : "email",
+                  autoComplete: `${usernameEnabled ? "username" : "email"}${passkey ? " webauthn" : ""}`,
                   className: classNames == null ? void 0 : classNames.input,
                   type: usernameEnabled ? "text" : "email",
                   placeholder: usernameEnabled ? localization.SIGN_IN_USERNAME_PLACEHOLDER : localization.EMAIL_PLACEHOLDER,
@@ -8472,7 +8473,7 @@ function SignInForm({
               /* @__PURE__ */ jsx75(FormControl, { children: /* @__PURE__ */ jsx75(
                 PasswordInput,
                 {
-                  autoComplete: "current-password",
+                  autoComplete: `current-password${passkey ? " webauthn" : ""}`,
                   className: classNames == null ? void 0 : classNames.input,
                   placeholder: localization.PASSWORD_PLACEHOLDER,
                   disabled: isSubmitting,
